exporters/etw/include/opentelemetry/exporters/etw/etw_tracer.h

```diff
@@ -985,8 +1011,9 @@ class TracerProvider : public opentelemetry::trace::TracerProvider
     UNREFERENCED_PARAMETER(args);
     UNREFERENCED_PARAMETER(schema_url);
     ETWProvider::EventFormat evtFmt = config_.encoding;
-    return nostd::shared_ptr<opentelemetry::trace::Tracer>{new (std::nothrow)
+    std::shared_ptr<opentelemetry::trace::Tracer> tracer{new (std::nothrow)
                                                                Tracer(*this, name, evtFmt)};
+    return nostd::shared_ptr<opentelemetry::trace::Tracer>{tracer};
```

Comment:
@ThomsonTan - Can you please review this PR when you have time? The changes were crashing earlier while creating Spans that are not sampled. The fix is to create the `Tracer` of `std::shared_ptr`, and then move/encapsulate it to `nostd::shared_ptr` (as done in line 1014 and 1016 above).